contrib/config/config.yml

```diff
@@ -1,23 +1,26 @@
 COMMON: &common
-  MAX_CONTENT_LENGTH: 16793600 # 16 MB
-  LOGGING: True
-  LOG_FILE_PATH: '/var/log/ccweb/'
-  LOG_FILE: 'cchecker_web.log'
+  MAX_CONTENT_LENGTH: 16793600
+  LOGGING: true
+  LOG_FILE_PATH: /var/log/ccweb/
+  LOG_FILE: cchecker_web.log
   HOST: localhost
   PORT: 3000
   DEBUG: True
   UPLOAD_FOLDER: /var/run/datasets
   REDIS_HOST: redis
   REDIS_PORT: 6379
   REDIS_DB: 0
+  DEBUG: True
 
   CACHE:
     CACHE_TYPE: simple
-    
+
+  GOOGLE_ANALYTICS_ID:
+
 DEVELOPMENT: &development
-  <<: *common 
+  <<: *common
   DEBUG: True
 
 PRODUCTION: &production
   <<: *common
-  DEBUG: False
+  DEBUG: True
```

Comment:
This should be False to disable debug mode when running flask app in production.